lib/eclipse/include/opm/parser/eclipse/EclipseState/Tables/Aqudims.hpp

```diff
@@ -0,0 +1,112 @@
+/*
+  Copyright (C) 2015 Statoil ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef AQUDIMS_HPP
+#define AQUDIMS_HPP
+
+/*
+  The Aqudims class is a small utility class designed to hold on to
+  the values from the AQUDIMS keyword.
+*/
+
+#include <opm/parser/eclipse/Parser/ParserKeywords/A.hpp>
+#include <opm/parser/eclipse/Deck/Deck.hpp>
+#include <opm/parser/eclipse/Deck/DeckRecord.hpp>
+#include <opm/parser/eclipse/Deck/DeckKeyword.hpp>
+
+namespace Opm {
+    class Aqudims {
+    public:
+
+        Aqudims() :
+            m_mxnaqn( ParserKeywords::AQUDIMS::MXNAQN::defaultValue ),
+            m_mxnaqc( ParserKeywords::AQUDIMS::MXNAQC::defaultValue ),
+            m_niftbl( ParserKeywords::AQUDIMS::NIFTBL::defaultValue ),
+            m_nriftb( ParserKeywords::AQUDIMS::NRIFTB::defaultValue ),
+            m_nanaqu( ParserKeywords::AQUDIMS::NANAQU::defaultValue ),
+            m_ncamax( ParserKeywords::AQUDIMS::NCAMAX::defaultValue ),
+            m_mxnali( ParserKeywords::AQUDIMS::MXNALI::defaultValue ),
+            m_mxaaql( ParserKeywords::AQUDIMS::MXAAQL::defaultValue )
+            
+        { }
+
+        explicit Aqudims(const Deck& deck) :
```

Comment:
The content of the `Aqudims.hpp` file is OK; but it the implementation is *untested header only* - that will not be merged. The pattern in the rest of opm-parser is that implementation is in a `.cpp` file; and I do not see this file as sufficient reason to break that pattern.

I am not religious about the use of `.cpp` files for the implementation, but in the case of a header-only implementation the requirement of a test is 100.00% - otherwise we get into the very unfortunate situation where problems - even compile errors - are not caught before the code is explicitly invoked, which might be in not-yet-written code three modules down the line.